src/routes/transactions/mappers/multisig-transactions/multisig-transaction-details.mapper.ts

```diff
@@ -0,0 +1,230 @@
+import { Inject, Injectable } from '@nestjs/common';
+import { isEmpty } from 'lodash';
+import { MultisigTransaction } from '../../../../domain/safe/entities/multisig-transaction.entity';
+import { Safe } from '../../../../domain/safe/entities/safe.entity';
+import { SafeRepository } from '../../../../domain/safe/safe.repository';
+import { ISafeRepository } from '../../../../domain/safe/safe.repository.interface';
+import { TokenRepository } from '../../../../domain/tokens/token.repository';
+import { ITokenRepository } from '../../../../domain/tokens/token.repository.interface';
+import {
+  LoggingService,
+  ILoggingService,
+} from '../../../../logging/logging.interface';
+import { AddressInfoHelper } from '../../../common/address-info/address-info.helper';
+import { NULL_ADDRESS } from '../../../common/constants';
+import { AddressInfo } from '../../../common/entities/address-info.entity';
+import {
+  MULTISIG_TRANSACTION_PREFIX,
+  TRANSACTION_ID_SEPARATOR,
+} from '../../constants';
+import { TransactionData } from '../../entities/transaction-data.entity';
+import {
+  MultisigConfirmationDetails,
+  MultisigExecutionDetails,
+} from '../../entities/transaction-details/multisig-execution-details.entity';
+import { TransactionDetails } from '../../entities/transaction-details/transaction-details.entity';
+import { TransactionStatus } from '../../entities/transaction-status.entity';
+import { SafeAppInfoMapper } from '../common/safe-app-info.mapper';
+import { TransactionDataMapper } from '../common/transaction-data.mapper';
+import { MultisigTransactionInfoMapper } from '../common/transaction-info.mapper';
+import { MultisigTransactionStatusMapper } from './multisig-transaction-status.mapper';
+
+@Injectable()
+export class MultisigTransactionDetailsMapper {
```

Comment:
We are not testing the fallback values for most of the properties returned by this mapper.

Given the size of the PR/tests involved we could think about having a different PR for it too. If you think this can be tackled easily from the route perspective tests then I'd go for it. We can also think about creating a unit test just for this mapper (as we have with other mappers).